dev-aws/otel/otel.tf

```diff
@@ -29,7 +29,13 @@ module "tempo_distributor" {
   consume_topics   = [kafka_topic.otlp_spans.name]
   consume_groups   = ["processor-tempo"]
   cert_common_name = "otel/tempo-distributor"
-  # Limit consuming to 10 MB/s / broker
-  consumer_byte_rate = 10485760
 }
 
+resource "kafka_quota" "tempo_distributor" {
```

Comment:
AFAIK we put this in place to NOT limit it to the default 5MB.

I think we should just remove it.

Reply:
> AFAIK we put this in place to NOT limit it to the default 5MB.
> 
> I think we should just remove it.

6e38961a108ae5a612e1e124dd7fb0cfe9419612